pyface/ui/qt4/code_editor/replace_widget.py

```diff
@@ -18,7 +18,7 @@
 
 class ReplaceWidget(FindWidget):
     def __init__(self, parent):
-        super(FindWidget, self).__init__(parent)
```

Comment:
Note this class is not `FindWidget`. (although I believe this will still work fine, just pointing it out.
Also, the only different between this `__init__` method and the `FindWidget` `__init__` method is the line `form_layout.addRow("Rep&lace", self._create_replace_control())`

Reply:
Ahh. Looks like this needs to stay as is.